emails/templates/welcome_on_signup.txt

```diff
@@ -0,0 +1,11 @@
+[[Subject .Subject "Welcome to Grafana"]]
+
+Hi [[.Name]],
+
+Welcome! Ready to start building some beautiful metric and analytic dashboards?
+
+If you are new to Grafana please read the Getting Started guid than can be found on https://grafana.com/docs/grafana/latest/getting-started/getting-started/.
+
+Thank you for joining our community.
```

Comment:
```suggestion
Thank-you for joining our community.
```

Reply:
@oddlittlebird This suggestion is incorrect English spelling.